pkgs/build-support/fetchsteam/default.nix

```
@@ -0,0 +1,56 @@
+{
+  lib,
+  runCommand,
+  depotdownloader,
+  cacert,
+  writeText,
+}:
+
+{
+  name ? "${toString app}-${toString depot}-${toString manifest}-depot",
+  app,
+  depot,
+  manifest,
+  branch ? null,
+  language ? null,
+  lowViolence ? false,
+  fileList ? [ ],
+  fileListRegex ? false,
+  debug ? false,
+  hash ? lib.fakeHash,
+}:
+
+runCommand name
+  {
+    nativeBuildInputs = [ depotdownloader ];
+
+    env.SSL_CERT_FILE = "${cacert}/etc/ssl/certs/ca-bundle.crt";
+
+    outputHash = hash;
+    outputHashAlgo = "sha256";
+    outputHashMode = "recursive";
+  }
+  ''
+    # Hack to prevent DepotDownloader from crashing trying to write to
+    # ~/.local/share/
+    export HOME
+    HOME=$(mktemp -d)
```

```suggestion
    HOME=$(mktemp -d)
    export HOME
```
I'm not sure what effect the current order of these commands would have? If anything I would think it would lead to more issues as the exported HOME is not the same as the one in the local scope